src/Hl7.Fhir.Base/Specification/Terminology/MimeTypeTerminologyService.cs

```diff
@@ -114,14 +115,15 @@ private async Task<Parameters> validateCodeVS(CodeableConcept cc)
         private static Task<Parameters> validateCodeVS(string? code, string? system)
         {
             var result = new Parameters();
+            var systemUri = system != null ? new Canonical(system)?.Uri : null;
```

Comment:
I think the `?` in `new Canonical(system)?.Uri` is not needed here: after construction it cannot be null...